backend/app/Http/Controllers/EmployeeController.php

```diff
@@ -291,7 +291,7 @@ public function deleteEmployee($employeeId): JsonResponse
 
             return $this->sendResponse(false, null, 'Employee deleted successfully', null, Response::HTTP_OK);
         } catch (Exception $e) {
-            return $this->sendResponse(true, "Could not fetch employee ", $e->getMessage(), null,  Response::HTTP_INTERNAL_SERVER_ERROR);
+            return $this->sendResponse(true, $e->getMessage(), "Could not fetch employee ", null,  Response::HTTP_INTERNAL_SERVER_ERROR);
```

Comment:
this is not written by you... so you can't change it.

Reply:
So I should leave some with message without error and error without message? Wow.

No wahala. I think say na my controller make I sanitize am. Obviously its yours. No issues

Reply:
lol... no ownership here. we all did this together... I am not the owner of anything...